docs/usage.md

```diff
@@ -2,577 +2,786 @@
 
-```bash
---input example_fastq.tsv
+```console
+patient,sex,status,sample,cram,crai
+patient1,XX,0,normal_sample,test_mapped.cram,test_mapped.cram.crai
+patient1,XX,1,tumor_sample,test2_mapped.cram,test2_mapped.cram.crai
+patient1,XX,1,relapse_sample,test3_mapped.cram,test3_mapped.cram.crai
 ```
 
-Or, for a normal/tumor pair:
+#### Start with annotation (`--step annotate`)
 
-In this example (`example_pair_fastq.tsv`), there are 3 read groups for the normal sample and 2 for the tumor sample.
+For starting from the annotation step, the CSV file must contain at least the columns `patient`, `sample`, `vcf`.
 
-| | | | | | | |
-|-|-|-|-|-|-|-|
-|SUBJECT_ID|XX|0|SAMPLE_ID1|1|/samples/normal1_1.fastq.gz|/samples/normal1_2.fastq.gz|
-|SUBJECT_ID|XX|0|SAMPLE_ID1|2|/samples/normal2_1.fastq.gz|/samples/normal2_2.fastq.gz|
-|SUBJECT_ID|XX|0|SAMPLE_ID1|3|/samples/normal3_1.fastq.gz|/samples/normal3_2.fastq.gz|
-|SUBJECT_ID|XX|1|SAMPLE_ID2|1|/samples/tumor1_1.fastq.gz|/samples/tumor1_2.fastq.gz|
-|SUBJECT_ID|XX|1|SAMPLE_ID2|2|/samples/tumor2_1.fastq.gz|/samples/tumor2_2.fastq.gz|
+As Sarek will use [bgzip](http://www.htslib.org/doc/bgzip.html) and [tabix](http://www.htslib.org/doc/tabix.html) to compress and index the annotated VCF files, it expects the input VCF files to be sorted and compressed.
 
-```bash
---input example_pair_fastq.tsv
+Example:
+
+```console
+patient,sample,vcf
+patient1,test_sample,test.vcf.gz
 ```
 
-#### --input &lt;uBAM&gt; --step mapping
+The Sarek-generated CSV file is stored under `results/csv/variantcalled.csv` and will automatically be used as an input when specifying the parameter `--step annotation`.
 
-The `TSV` file to start with the mapping step (`--step mapping`) with `unmapped BAM` files should contain the columns:
+##### Full samplesheet
 
-`subject sex status sample lane bam`
+In this example, all possible columns are used including the `variantcaller` information per sample:
 
-In this example (`example_ubam.tsv`), there are 3 read groups.
+```console
+patient,sample,variantcaller,vcf
+test,sample3,strelka,sample3.variants.vcf.gz
+test,sample4_vs_sample3,manta,sample4_vs_sample3.diploid_sv.vcf.gz
+test,sample4_vs_sample3,manta,sample4_vs_sample3.somatic_sv.vcf.gz
+```
 
-| | | | | | |
-|-|-|-|-|-|-|
-|SUBJECT_ID|XX|0|SAMPLE_ID|1|/samples/normal_1.bam|
-|SUBJECT_ID|XX|0|SAMPLE_ID|2|/samples/normal_2.bam|
-|SUBJECT_ID|XX|0|SAMPLE_ID|3|/samples/normal_3.bam|
+### Updating the pipeline
 
-```bash
---input example_ubam.tsv
+When you run the above command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
```

Comment:
```suggestion
When you run the below command, Nextflow automatically pulls the pipeline code from GitHub and stores it as a cached version. When running the pipeline after this, it will always use the cached version if available - even if the pipeline has been updated since. To make sure that you're running the latest version of the pipeline, make sure that you regularly update the cached version of the pipeline:
```